src/coreclr/jit/instr.h

```diff
@@ -148,6 +160,13 @@ enum insFlags: unsigned
     INS_FLAGS_SET = 0x01,
     INS_FLAGS_DONT_CARE = 0x02,
 };
+#elif defined(TARGET_LOONGARCH64)
+enum insFlags: unsigned
+{
+    INS_FLAGS_NOT_SET = 0x00,
+    INS_FLAGS_SET = 0x01,
+    INS_FLAGS_DONT_CARE = 0x02,
+};
```

Comment:
This looks exactly the same as for ARM/ARM64 above. Is it actually used for LoongArch64? If not, remove it. If it is, just add `|| defined(TARGET_LOONGARCH64)` above.

Or do you expect it to change? Even then, if you need to change it, you can split the definition later.

Reply:
OK, Thanks